api/admin.go

```diff
@@ -195,6 +195,15 @@ func testEmail(c *Context, w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	// if the user hasn't changed their email settings, fill in the actual SMTP password so that
+	// the user can verify an existing SMTP connection
+	if cfg.EmailSettings.SMTPServer == utils.Cfg.EmailSettings.SMTPServer &&
```

Comment:
I was worried about someone with access to the system console pointing it to some other address that could snoop the password. I'm not sure how much that's actually a concern though so I'll queue it for the dev meeting